### PR TITLE
renamed sampling to selection in lightly external

### PR DESCRIPTION
**Pull request by MalteEbner**
## Description
Renamed in /lightly, /tests and /docs

## Tests
Unittests are running through.
Unmocked tests are running through.
Docs with `make html-noplot` were tested and raises the same 3 errors as without this PR, see https://github.com/lightly-ai/lightly/pull/749

Docs with `make html` were NOT tested.

### CODE REVIEW

**Comment by philippmwirth** [comment]
Sorry, I thought this wasn't ready for review yet. How is this ready for review if building the docs fails?

**Comment by MalteEbner** [comment]
> Sorry, I thought this wasn't ready for review yet. How is this ready for review if building the docs fails?

I just tested with the develop branch, which causes the same errors. Thus this is not about this PR, but has existed before. I created an issue to fix it on the develop branch and made it blocking this.
https://linear.app/lightly/issue/LIG-890/fix-docs-creation-with-html-noplot

File: docs/source/docker/overview.rst [hunk 1]
```diff
@@ -27,18 +27,18 @@ We worked hard to make this happen and are very proud to present you with the fo
 
   * Supported object categories are: bicycle, bus, car, motorcycle, person, train, truck
 
-* Sample more than 1 Million samples within a few hours!
+* Select more than 1 Million samples within a few hours!
```
Comment (philippmwirth) [inline docs/source/docker/overview.rst:30]: This doesn't make sense. It's to select *from* more than 1 million samples

File: docs/source/getting_started/active_learning.rst [hunk 2]
```diff
@@ -101,23 +101,23 @@ Then, in your Python script, you will need to initialize the `ApiWorkflowClient`
    and tell the `ActiveLearningAgent` to only sample from this tag. To do so, set
    the `query_tag_name` argument in the constructor of the agent.
 
-Let's configure the sampling request and request an initial selection next:
+Let's configure the selection request and request an initial selection next:
```
Comment (philippmwirth) [inline docs/source/getting_started/active_learning.rst:104]: there's a repetition of words here now so we should probably rewrite this sentence.

File: docs/source/getting_started/active_learning.rst [hunk 3]
```diff
@@ -101,23 +101,23 @@ Then, in your Python script, you will need to initialize the `ApiWorkflowClient`
...
 
 .. code-block:: Python
 
-   from lightly.active_learning.config import SamplerConfig
+   from lightly.active_learning.config import SelectionConfig
    from lightly.openapi_generated.swagger_client import SamplingMethod
 
    # we want an initial pool of 150 images
-   config = SamplerConfig(n_samples=150, method=SamplingMethod.CORESET, name='initial-selection')
+   config = SelectionConfig(n_samples=150, method=SamplingMethod.CORESET, name='initial-selection')
```
Comment (philippmwirth) [inline docs/source/getting_started/active_learning.rst:112]: this is now really ugly because it's inconsistent :/ we always talk about selection strategy and all of the sudden it's `method=SamplingMethod.CORESET`

File: docs/source/getting_started/active_learning.rst [hunk 4]
```diff
@@ -172,9 +172,9 @@ here is that the argument `n_samples` always refers to the total size of the lab
 .. code-block:: Python
 
    # we want a total of 200 images after the first iteration (50 new samples)
-   # this time, we use the CORAL sampler and provide a scorer to the query
-   config = SamplerConfig(n_samples=200, method=SamplingMethod.CORAL, name='al-iteration-1')
-   al_agent.query(sampler_config, scorer)
+   # this time, we use the CORAL selection strategy and provide a scorer to the query
+   selection_config = SelectionConfig(n_samples=200, method=SamplingMethod.CORAL, name='al-iteration-1')
```
Comment (philippmwirth) [inline docs/source/getting_started/active_learning.rst:176]: same here with the `method=...`

File: docs/source/getting_started/platform.rst [hunk 5]
```diff
@@ -264,42 +264,42 @@ you can use a simple script to translate it to the coco format:
...
 
-#. Give a name to your subsampling so that you can later compare 
-   the different samplings.
+#. Give a name to your selectiong so that you can later compare
+   the different selections.
 
-#. Hit "Process" to start sampling the data. Each sample is now assigned an 
-   "importance score". The exact meaning of the score depends on the sampler.
+#. Hit "Process" to start selecting the data. Each sample is now assigned an
+   "importance score". The exact meaning of the score depends on the selection strategy.
```
Comment (philippmwirth) [inline docs/source/getting_started/platform.rst:288]: Maybe we can leave " The exact meaning of the score depends on the selection strategy." away

File: docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 .. note:: 
     In many real world applications of machine learning datasets are heavily biased.
-    This is a direct consequence of the fact that the data is sampled from a distribution
+    This is a direct consequence of the fact that the data is selected from a distribution
```
Comment (philippmwirth) [inline docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py:5]: here it's not necessary to replace the word sampled

File: docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py [hunk 7]
```diff
@@ -188,8 +188,8 @@
 
     Scatter plot of the tag we named "Fish".
 
-Now, we can use CORESET sampling to diversify this tag and reduce the number of images in it (see `Tutorial 2: Diversify the Sunflowers Dataset` 
-if you're not familiar with subsampling). We use CORESET to create the tag `FewerFish` with only 10 remaining images.
+Now, we can use CORESET selection strategy to diversify this tag and reduce the number of images in it (see `Tutorial 2: Diversify the Sunflowers Dataset`
+if you're not familiar with selection). We use CORESET to create the tag `FewerFish` with only 10 remaining images.
```
Comment (philippmwirth) [inline docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py:192]: If you're not familiar with selection sounds strange I would rephrase this. Maybe something like "See tutorial xy for more information on selection strategies".

File: docs/source/tutorials_source/platform/tutorial_sunflowers.py [hunk 8]
```diff
@@ -5,8 +5,8 @@
 Tutorial 2: Diversify the Sunflowers Dataset
 =============================================
 
-This tutorial highlights the basic functionality of sampling in the web-app.
-You can use the Coreset sampling algorithm to choose a diverse subset of your dataset.
+This tutorial highlights the basic functionality of selecting in the web-app.
```
Comment (philippmwirth) [inline docs/source/tutorials_source/platform/tutorial_sunflowers.py:8]: selecting what? I don't think we can use selecting just as a plain replacement of sampling. It sounds very unnatural. Selecting a subset, selecting images, etc are alternatives.

File: docs/source/tutorials_source/platform/tutorial_sunflowers.py [hunk 9]
```diff
@@ -77,48 +77,48 @@
    You can switch between the PCA, tSNE and UMAP dimensionality reduction methods.
 
 Right above the scatter plot you should see a button "Create Sampling". Click on it to
-create a sampling. You will need to configure the following settings:
+create a selection. You will need to configure the following settings:
 
-* **Embedding:** Choose the embedding to use for the sampling.
-* **Sampling Strategy:** Choose the sampling strategy to use. This will be one of:
+* **Embedding:** Choose the embedding to use for the selection.
+* **Sampling Strategy:** Choose the selection strategy to use. This will be one of:
```
Comment (philippmwirth) [inline docs/source/tutorials_source/platform/tutorial_sunflowers.py:83]: sampling strategy?

File: lightly/active_learning/agents/agent.py [hunk 10]
```diff
@@ -195,28 +195,28 @@ def upload_scores(self, al_scorer: Scorer):
...
 
     def query(self,
-              sampler_config: SamplerConfig,
+              selection_config: SelectionConfig,
               al_scorer: Scorer = None):
         """Performs an active learning query.
 
         First the active learning scores are computed and uploaded,
-        then the sampling query is performed.
+        then the selection query is performed.
```
Comment (philippmwirth) [inline lightly/active_learning/agents/agent.py:203]: maybe just use query instead of selection query

File: lightly/api/api_workflow_selection.py [hunk 11]
```diff
@@ -52,22 +52,20 @@ def upload_scores(self, al_scores: Dict[str, np.ndarray], query_tag_id: str = No
                 tag_id=query_tag_id,
             )
 
-    def sampling(self, sampler_config: SamplerConfig, preselected_tag_id: str = None, query_tag_id: str = None) \
+    def selection(self, selection_config: SelectionConfig, preselected_tag_id: str = None, query_tag_id: str = None) \
```
Comment (philippmwirth) [inline lightly/api/api_workflow_selection.py:55]: this is a breaking change. If we really want to do this now we should keep the old method and raise a deprecation warnign.

File: lightly/api/api_workflow_selection.py [hunk 12]
```diff
@@ -127,26 +125,26 @@ def sampling(self, sampler_config: SamplerConfig, preselected_tag_id: str = None
 
         return new_tag_data
 
-    def _create_sampling_create_request(self, sampler_config: SamplerConfig, preselected_tag_id: str, query_tag_id: str
+    def _create_sampling_create_request(self, selection_config: SelectionConfig, preselected_tag_id: str, query_tag_id: str
```
Comment (philippmwirth) [inline lightly/api/api_workflow_selection.py:128]: rename to `selection`

**Review by philippmwirth — CHANGES_REQUESTED** [review]
As I said before I have doubts that we should introduce these big breaking changes right now. I would first get the opinions from @IgorSusmelj @guarin @japrescott.

If we really go with it we should keep the old functions around and raise deprecation warnings before we actually remove them.

File: docs/source/docker/advanced/active_learning.rst [hunk 13]
```diff
@@ -194,10 +194,10 @@ Run Active Learning using the Docker
 At this point you should have an `embeddings.csv` file with the active learning 
 scores in a column named `al_scores`. 
 
-We can now perform an active learning sampling using the `CORAL` sampler. In order
-to do the sampling on the `embeddings.csv` file we need to make this file
+We can now perform an active learning selection using the `CORAL` selection strategy. In order
+to do the selection on the `embeddings.csv` file we need to make this file
 accessible to the docker. We can do this by using the `shared_dir` feature of the
-docker as described in :ref:`docker-sampling-from-embeddings`. 
+docker as described in :ref:`docker-selection-from-embeddings`.
```
[the remark this answers is not in the document]
Comment (MalteEbner) [reply docs/source/docker/advanced/active_learning.rst:200]: I switched back to the old ref to prevent this.

File: docs/source/docker/getting_started/first_steps.rst [hunk 14]
```diff
@@ -284,9 +284,9 @@ do so.
 A list of all input parameters can be found here: :ref:`rst-docker-parameters`
 
 
-.. _docker-sampling-from-embeddings:
+.. _docker-selection-from-embeddings:
```
[the remark this answers is not in the document]
Comment (MalteEbner) [reply docs/source/docker/getting_started/first_steps.rst:287]: see https://github.com/lightly-ai/lightly/pull/735#discussion_r839569999

File: lightly/api/api_workflow_selection.py (hunk 11 above)
Comment (MalteEbner) [reply lightly/api/api_workflow_selection.py:55]: I kept the old method (just calling the new one) and added a PendingDeprecationWarning. A unittest checks for it.

**Review by philippmwirth — CHANGES_REQUESTED** [review]
Some more inputs. The search and replace doesn't really work in this case I think (see my comments).

Some of my previous comments haven't been addressed yet. Did you see them? I think you might need to unfold them (XY comments were hidden).

File: docs/source/getting_started/active_learning.rst (hunk 3 above)
Comment (MalteEbner) [reply docs/source/getting_started/active_learning.rst:112]: This will need to be changed with https://linear.app/lightly/issue/LIG-614/rename-sampling-to-selection-internal-in-pip-package-code

File: docs/source/getting_started/active_learning.rst (hunk 4 above)
Comment (MalteEbner) [reply docs/source/getting_started/active_learning.rst:176]: See https://github.com/lightly-ai/lightly/pull/735#discussion_r841432558

File: docs/source/docker/examples/datasets_in_the_wild.rst [hunk 15]
```diff
@@ -162,7 +162,7 @@ Let's have a look at some statistics of the two obtained datasets:
 We notice the following when looking at this table:
 
 - The **min distance** between two samples was 0 after ffmpeg selection whereas the
-  min distance significantly increased using coreset sampling.
+  min distance significantly increased using CORESET selection strategy.
```
[the remark this answers is not in the document]
Comment (MalteEbner) [reply docs/source/docker/examples/datasets_in_the_wild.rst:165]: We now agreed to use CORESET/CORAL/RANDOM everywhere.

File: docs/source/docker/advanced/active_learning.rst [hunk 16]
```diff
@@ -23,8 +23,8 @@ You can create embeddings using your own model. Just make sure the resulting
 
 Alternatively, you can run the docker as usual and as described in the 
 :ref:`rst-docker-first-steps` section.
-The only difference is that you set the number of samples to be sampled to `1.0`,
-as this prevents sampling and just creates the embedding of the full dataset.
+The only difference is that you set the number of samples to be selected to 1.0,
+as this simply creates an embedding of the full dataset"t.
```
Comment (philippmwirth) [inline docs/source/docker/advanced/active_learning.rst:27]: typo

File: docs/source/docker/advanced/active_learning.rst [hunk 17]
```diff
@@ -194,10 +194,10 @@ Run Active Learning using the Docker
 At this point you should have an `embeddings.csv` file with the active learning 
 scores in a column named `al_scores`. 
 
-We can now perform an active learning sampling using the `CORAL` sampler. In order
-to do the sampling on the `embeddings.csv` file we need to make this file
+We can now perform an active learning selection using the `CORAL` selection strategy. In order
```
[the remark this answers is not in the document]
Comment (philippmwirth) [reply docs/source/docker/advanced/active_learning.rst:197]: @MalteEbner you gave me a 👍 but didn't change anything, did you miss this one?

File: docs/source/docker/examples/datasets_in_the_wild.rst [hunk 18]
```diff
@@ -162,20 +162,19 @@ Let's have a look at some statistics of the two obtained datasets:
...
 - The **min distance** between two samples was 0 after ffmpeg selection whereas the
-  min distance significantly increased using coreset sampling.
+  min distance significantly increased using coreset selection strategy.
 
   - 0 distance means that there are at least two samples completely identical
     (e.g. two frames in the video are the same)
 
 - The **mean distance** between the original dataset, ffmpeg, as well as 
-  random selection, is very similar. The coreset selection however differs 
+  RANDOM selection, is very similar. The coreset selection however differs
```
Comment (philippmwirth) [inline docs/source/docker/examples/datasets_in_the_wild.rst:171]: why is RANDOM all caps and coreset in lower case? I'm confused now 😄

File: docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py (hunk 7 above)
Comment (philippmwirth) [reply docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py:192]: Here too, you gave the 👍 but didn't make any changes

File: docs/source/tutorials_source/platform/tutorial_sunflowers.py (hunk 8 above)
Comment (philippmwirth) [reply docs/source/tutorials_source/platform/tutorial_sunflowers.py:8]: Here too, you didn't make any changes. Did you change things for my other comments?

**Review by philippmwirth — CHANGES_REQUESTED** [review]

File: docs/source/docker/advanced/active_learning.rst [hunk 19]
```diff
@@ -194,10 +194,10 @@ Run Active Learning using the Docker
 At this point you should have an `embeddings.csv` file with the active learning 
 scores in a column named `al_scores`. 
 
-We can now perform an active learning sampling using the `CORAL` sampler. In order
-to do the sampling on the `embeddings.csv` file we need to make this file
+We can now perform an active learning iteration using the `coral` selection strategy. In order
```
Comment (japrescott) [inline docs/source/docker/advanced/active_learning.rst:197]: we said CORAL, no?

File: docs/source/docker/advanced/meta_information.rst [hunk 20]
```diff
@@ -89,7 +89,7 @@ additional samples that are enriching the existing selection.
 
 .. note:: Pre-selected samples also count for the target number of samples.
           For example, you have a dataset with 100 samples. If you pre-select
-          60 and want to sample 50, sampling would have no effect since there
+          60 and want to sample 50, selecting would have no effect since there
```
Comment (japrescott) [inline docs/source/docker/advanced/meta_information.rst:92]: Its confusing because of the pre-select;
selecting -> your selection strategy

File: docs/source/docker/configuration/configuration.rst [hunk 21]
```diff
@@ -105,13 +105,13 @@ The following are parameters which can be passed to the container:
...
 
-  # sampling
+  # Selection
   # Choose from ['coreset', 'random'].
   method: 'coreset'
   stopping_condition:
     # Float in [0., 1.] for percentage, int for number of samples, -1 means inactive.
     n_samples: -1    
-    # Float, minimum distance between two images in the sampled dataset, -1. means inactive.           
+    # Float, minimum distance between two images in the selected dataset, -1. means inactive.
```
Comment (japrescott) [inline docs/source/docker/configuration/configuration.rst:114]: in the selected dataset -> in the dataset you are applying the selection strategy

File: docs/source/docker/examples/datasets_in_the_wild.rst [hunk 22]
```diff
@@ -162,20 +162,19 @@ Let's have a look at some statistics of the two obtained datasets:
...
-  random selection, is very similar. The coreset selection however differs 
+  random selection, is very similar. The coreset selection however differs
   significantly with a higher mean (higher diversity) in the selected dataset.
 
 - The **10th percentile** shows similar behavior to the mean distance.
 
 As you see in this example just selecting every N-th frame is similar to
-selecting frames randomly. More sophisticated selection methods, such as 
-coreset sampling which has been optimized for Lightly Docker, result in 
+selecting frames randomly. More sophisticated selection methods, such as the coreset which has been optimized for Lightly Docker, result in
```
Comment (japrescott) [inline docs/source/docker/examples/datasets_in_the_wild.rst:177]: selection methods -> selection strategies

File: docs/source/docker/getting_started/first_steps.rst [hunk 23]
```diff
@@ -168,8 +168,8 @@ Then you can use the following docker run command and the Lightly docker will on
 
 
 
-Embedding and Sampling a Dataset
------------------------------------
+Embedding a Dataset and Selecting from it
```
Comment (japrescott) [inline docs/source/docker/getting_started/first_steps.rst:171]: english does not have caps

Comment (japrescott) [reply docs/source/docker/getting_started/first_steps.rst:171]: but it seems its wrong everywhere... leave it..

File: docs/source/docker/getting_started/first_steps.rst [hunk 24]
```diff
@@ -449,7 +449,7 @@ To facilitate sustainability and reproducibility in ML, the docker container
 has an integrated reporting component. For every dataset, you run through the container
 an output directory gets created with the exact configuration used for the experiment. 
 Additionally, plots, statistics, and more information collected either during the training of the
-self-supervised model, embedding, or sampling of the dataset are provided. 
+self-supervised model, embedding, or selection of the dataset are provided.
```
Comment (japrescott) [inline docs/source/docker/getting_started/first_steps.rst:452]: selection of the dataset -> selection within the dataset

File: docs/source/docker/getting_started/first_steps.rst [hunk 25]
```diff
@@ -509,7 +509,7 @@ The output directory is structured in the following way:
 
    * information about the job (duration, processed files etc.)
    * estimated savings in terms of labeling costs and CO2 due to the smaller dataset
-   * statistics about the dataset before and after sampling
+   * statistics about the dataset before and after selection
```
Comment (japrescott) [inline docs/source/docker/getting_started/first_steps.rst:512]: and after selection -> and after applying the selection strategy

File: docs/source/docker/getting_started/first_steps.rst [hunk 26]
```diff
@@ -566,12 +566,12 @@ Below you find a typical output folder structure.
...
 
 
-Evaluation of the Sampling Proces
+Evaluation of the Selection Proces
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 **Histograms and Plots**
 
-The report contains histograms of the pairwise distance between images before and after the sampling.
+The report contains histograms of the pairwise distance between images before and after the selection.
```
Comment (japrescott) [inline docs/source/docker/getting_started/first_steps.rst:574]: and after the selection -> and after applying the selection strategy

File: docs/source/docker/integration/docker_trigger_from_api.rst [hunk 27]
```diff
@@ -194,16 +194,16 @@ To see the progress of your docker run, go to the Lightly Platform and head to
 .. image:: ../getting_started/images/docker_runs_overview.png
 
 
-Use your subsampled dataset
+Use your selected dataset
```
Comment (japrescott) [inline docs/source/docker/integration/docker_trigger_from_api.rst:197]: this reads weird. Maybe its just me..

File: docs/source/docker/integration/docker_trigger_from_api.rst [hunk 28]
```diff
@@ -194,16 +194,16 @@ To see the progress of your docker run, go to the Lightly Platform and head to
...
 ---------------------------
 
-Once the docker run has finished, you can see your subsampled dataset in the 
+Once the docker run has finished, you can see your selected dataset in the
```
Comment (japrescott) [inline docs/source/docker/integration/docker_trigger_from_api.rst:200]: see your selected dataset in -> see your dataset with the selected images in

File: docs/source/docker/integration/docker_with_datasource.rst [hunk 29]
```diff
@@ -124,8 +124,8 @@ It remembers which raw data in your S3 bucket has already been processed
 and will ignore it in future docker runs.
 Thus you can run the docker with the same command again. It will find
 your new raw data in the S3 bucket, stream, embed and subsample it and then add it to
-your existing dataset. The samplers will take the existing data in your dataset
-into account when sampling new data to be added to your dataset.
+your existing dataset. The selection will take the existing data in your dataset
```
Comment (japrescott) [inline docs/source/docker/integration/docker_with_datasource.rst:127]: the selection strategy will

File: docs/source/docker/integration/docker_with_datasource.rst [hunk 30]
```diff
@@ -94,15 +94,15 @@ head to `My Docker Runs <https://app.lightly.ai/docker/runs>`_
 
 .. image:: ../getting_started/images/docker_runs_overview.png
 
-Use your subsampled dataset
+Use your selected dataset
 ---------------------------
 
-Once the docker run has finished, you can see your subsampled dataset in the Lightly Platform:
+Once the docker run has finished, you can see your selected dataset in the Lightly Platform:
```
Comment (japrescott) [inline docs/source/docker/integration/docker_with_datasource.rst:100]: see your selected dataset in -> see your dataset with the selected images in

File: docs/source/getting_started/active_learning.rst [hunk 31]
```diff
@@ -41,21 +41,21 @@ Lightly makes use of the following concepts for active learning:
...
+   The `SelectionConfig` allows the configuration of a selection request. In particular,
    you can set number of samples, the name of the resulting selection, and the `SamplingMethod`.
    Currently, you can set the `SamplingMethod` to one of the following:
 
-   * Random: Selects samples uniformly at random.
-   * Coreset: Greedily selects samples which are diverse.
-   * Coral: Combines Coreset with scores to do active learning.
+   * RANDOM: Selects samples uniformly at random.
+   * CORESET: Greedily selects samples which are diverse.
+   * CORAL: Combines CORESET with scores to do active learning.
```
Comment (japrescott) [inline docs/source/getting_started/active_learning.rst:51]: small or big?! further up its always small.. now its big again?

File: docs/source/docker/advanced/active_learning.rst (hunk 19 above)
Comment (MalteEbner) [reply docs/source/docker/advanced/active_learning.rst:197]: https://lightly-ai.slack.com/archives/C014FJT529M/p1649231912686499

File: docs/source/docker/integration/docker_trigger_from_api.rst (hunk 27 above)
Comment (MalteEbner) [reply docs/source/docker/integration/docker_trigger_from_api.rst:137]: We use the term "selected dataset" at many locations:
image

File: docs/source/docker/integration/docker_trigger_from_api.rst (hunk 28 above)
Comment (MalteEbner) [reply docs/source/docker/integration/docker_trigger_from_api.rst:140]: see https://github.com/lightly-ai/lightly/pull/735#discussion_r845343726

File: docs/source/docker/integration/docker_with_datasource.rst (hunk 30 above)
Comment (MalteEbner) [reply docs/source/docker/integration/docker_with_datasource.rst:108]: see https://github.com/lightly-ai/lightly/pull/735#discussion_r845343726

File: docs/source/getting_started/active_learning.rst (hunk 31 above)
Comment (MalteEbner) [reply docs/source/getting_started/active_learning.rst:51]: Always BIG in the normal docs, Small in the docker docs.

File: docs/source/docker/getting_started/first_steps.rst (hunk 25 above)
Comment (MalteEbner) [reply docs/source/docker/getting_started/first_steps.rst]: I used `before/after selection process` for these cases.

File: docs/source/docker/integration/docker_trigger_from_api.rst (hunk 27 above)
Comment (japrescott) [reply docs/source/docker/integration/docker_trigger_from_api.rst:137]: yiäh... but you gét my point? :) béforé it was sampléd datasét (which is kinda cléar what it méans) and now its séléctéd datasét which could bé 1) thé datasét I havé chosén (of many othérs) or 2) thé datasét that I want to do sométhing with. But not what it actually méans: "Thé datasét on which wé havé appliéd our séléction statégy".
Maybé wé can usé thé word "Usé your réducéd datasét" or "usé your optimizd datasét" or moré simply "thé yiéldéd datasét"? Maybé @IgorSusmelj and @philippmwirth know a béttér wording?

File: docs/source/docker/configuration/configuration.rst [hunk 32]
```diff
@@ -105,13 +105,15 @@ The following are parameters which can be passed to the container:
...
+  # Selection
   # Choose from ['coreset', 'random'].
   method: 'coreset'
+  # Choose when to stop the selection
   stopping_condition:
+    # The maximum number of samples selected
     # Float in [0., 1.] for percentage, int for number of samples, -1 means inactive.
     n_samples: -1    
-    # Float, minimum distance between two images in the sampled dataset, -1. means inactive.           
+    # Float, minimum distance between two images selected, -1. means inactive.
```
Comment (japrescott) [inline docs/source/docker/configuration/configuration.rst:116]: images selected -> selected images

File: docs/source/docker/examples/datasets_in_the_wild.rst [hunk 33]
```diff
@@ -162,21 +162,20 @@ Let's have a look at some statistics of the two obtained datasets:
 We notice the following when looking at this table:
 
 - The **min distance** between two samples was 0 after ffmpeg selection whereas the
-  min distance significantly increased using coreset sampling.
+  min distance significantly increased using coreset selection strategy.
```
Comment (japrescott) [inline docs/source/docker/examples/datasets_in_the_wild.rst:165]: put coreset in to ``

**Review by japrescott — COMMENTED** [review]
I am still super confused.. everything docker is always `coreset` and everything else is always CORESET? I thought we will call it uniform (except for where coreset is passed to the docker config).
Also, I think we always should highlight the `coreset` `coral` `CORESET` etc..

**Comment by japrescott** [comment]
maybe we also just merge this issue and redo formatting etc when we move to the news docs..

File: docs/source/docker/advanced/sequence_selection.rst [hunk 34]
```diff
@@ -17,7 +17,7 @@ Sequence selection consists of the following steps:
 
 1. Each input video is split into sequences of length `selected_sequence_length`.
 2. Next, the embeddings of all frames in a sequence are aggregated (averaged).
-3. The Lightly sampling algorithm selects relevant sequences.
+3. The Lightly selection strategy selects relevant sequences.
```
[the remark this answers is not in the document]
Comment (MalteEbner) [reply docs/source/docker/advanced/sequence_selection.rst:20]: I changed the sentence to `The selection is performed on sequence level.`

File: docs/source/docker/examples/datasets_in_the_wild.rst (hunk 33 above)
Comment (MalteEbner) [reply docs/source/docker/examples/datasets_in_the_wild.rst:165]: See https://github.com/lightly-ai/lightly/pull/735#discussion_r847111684

**Review by japrescott — APPROVED** [review]

File: docs/source/getting_started/platform.rst [hunk 35]
```diff
@@ -264,42 +264,42 @@ you can use a simple script to translate it to the coco format:
...
 section by clicking on it.
 
 #. Navigate to *"Analyze & Filter"* → *"Sampling"* through the menu on the left.
 
-#. Choose the embedding and sampling strategy to use for this sampling run.
+#. Choose the embedding and selection strategy to use for this selection.
 
-#. Give a name to your subsampling so that you can later compare 
-   the different samplings.
+#. Give a name to your selectiong so that you can later compare
```
Comment (philippmwirth) [inline docs/source/getting_started/platform.rst:284]: typo

**Review by philippmwirth — CHANGES_REQUESTED** [review]
1 more typo

**Review by philippmwirth — APPROVED** [review]
